plugins/services/src/js/components/SDKPlansScreen.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react";
+
+import { Service } from "#PLUGINS/services/src/js/types/Service";
+import SDKPlanTable from "#PLUGINS/services/src/js/components/SDKPlanTable";
+import { formatServicePlanStatus } from "#PLUGINS/services/src/js/types/ServicePlanStatus";
+import { ServicePlan } from "#PLUGINS/services/src/js/types/ServicePlan";
+
+export interface SDKPlansScreenProps {
+  service: Service;
+  plan: string;
+  handleSelectPlan: (name: string) => void;
+}
+
+class SDKPlansScreen extends React.PureComponent<SDKPlansScreenProps, {}> {
```

Comment:
I believe that should be either `Page` or `Tab`

Reply:
Tab would be fitting, 👍